content/sensu-core/2.0/reference/agent.md

```diff
@@ -0,0 +1,822 @@
+      --redact string                       comma-delimited customized list of fields to redact
+      --socket-host string                  address to bind the Sensu client socket to (default "127.0.0.1")
+      --socket-port int                     port the Sensu client socket listens on (default 3030)
+      --statsd-disable                      disables the statsd listener and metrics server
+      --statsd-event-handlers stringSlice   comma-delimited list of event handlers for statsd metrics
+      --statsd-flush-interval int           number of seconds between statsd flush (default 10)
+      --statsd-metrics-host string          address used for the statsd metrics server (default "127.0.0.1")
+      --statsd-metrics-port int             port used for the statsd metrics server (default 8125)
+      --subscriptions string                comma-delimited list of agent subscriptions
+      --user string                         agent user (default "agent")
+{{< /highlight >}}
+
+### General configuration flags
+
+| backend-url |      |
+--------------|------
+description   | ws or wss URL of the Sensu backend server. To specify multiple backends using `sensu-agent start`, use this flag multiple times.
+type          | String
+default       | `ws://127.0.0.1:8081`
+example       | {{< highlight shell >}}# Command line example
+sensu-agent start --backend-url ws://0.0.0.0:8081
+
+# /etc/sensu/agent.yml example
+backend-url:
+  - "ws://0.0.0.0:8081"{{< /highlight >}}
+
+<a name="cache-dir">
+
+| cache-dir   |      |
+--------------|------
+description   | Path to store cached data
+type          | String
+default       | <ul><li>Linux: `/var/cache/sensu/sensu-agent`</li><li>Windows: `C:\\ProgramData\sensu\cache\sensu-agent`</li></ul>
+example       | {{< highlight shell >}}# Command line example
+sensu-agent start --cache-dir /cache/sensu-agent
+
+# /etc/sensu/agent.yml example
+cache-dir: "/cache/sensu-agent"{{< /highlight >}}
+
+
+| config-file |      |
+--------------|------
+description   | Path to Sensu agent config file
+type          | String
+default       | <ul><li>Linux: `/etc/sensu/agent.yml`</li><li>FreeBSD: `/usr/local/etc/sensu/agent.yml`</li><li>Windows: `C:\\ProgramData\sensu\config\agent.yml`</li></ul>
+example       | {{< highlight shell >}}# Command line example
+sensu-agent start --config-file /sensu/agent.yml
+sensu-agent start --c /sensu/agent.yml
+
+# /etc/sensu/agent.yml example
+config-file: "/sensu/agent.yml"{{< /highlight >}}
+
+
+| extended-attributes |      |
+----------------------|------
+description           | Extended attributes to include in the agent entity in serialized JSON format
+type                  | Serialized JSON object
+example               | {{< highlight shell >}}# Command line example
+sensu-agent start --extended-attributes '{"team":"ops"}'
+
+# /etc/sensu/agent.yml example
+extended-attributes: "{\"team\":\"ops\"}"{{< /highlight >}}
```

Comment:
Based on the difficulties Melissa had yesterday, it would probably safer to do this instead:

`extended-attributes: '{"team":"ops"}'`